src/style.css

```
@@ -0,0 +1,67 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+.form-wrapper {
+  width: 100%;
+  display: flex;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  padding: 0;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.form-wrapper:focus {
+  border-radius: 3px;
+}
+
+.first-wrapper {
+  margin: 20px 0;
+  height: 70px;
+}
+
+.second-wrapper {
+  width: 100%;
+  height: 42px;
+}
+
+.search {
+  width: 100%;
+  border: 0;
+  font-family: Avenir, sans-serif;
+  padding: 0;
+  color: rgba(61, 78, 97, 1);
+}
+
+.searchOne {
+  font-size: 16px;
+}
+
+.searchTwo {
+  font-size: 14px;
+}
+
+.search:focus {
+  font-weight: bold;
+  border: 0;
+  outline-width: 0;
+  text-shadow: 0 3px 3px rgba(61, 78, 97, 0.3);
+}
+
+.form-wrapper:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-image {
+  object-fit: none;
+}
+
+label {
+  display: flex;
+}
+
+input:focus::placeholder {
+  font-weight: 300;
```

placeholder should not have any styles, but shadow is still visible. Add
```suggestion
  font-weight: 300;
  text-shadow: none
```